src/frontends/tensorflow_common/src/op/sparse_segment_mean.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/embedding_segments_sum.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/scatter_update.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/unique.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_sparse_segment_mean_op(const NodeContext& node) {
+    default_op_checks(node, 3, {"SparseSegmentMean"});
+    auto data = node.get_input(0);
+    auto indices = node.get_input(1);
+    auto segment_ids = node.get_input(2);
+
+    // get the last index from segment_ids
+    auto segments_ids_size = std::make_shared<v3::ShapeOf>(segment_ids, indices.get_element_type());
```

Comment:
`indices.get_element_type()` can be undefined at this stage. Please use concrete time `i64`

Reply:
propose to convert both `indices` and `segment_ids` to `i64` and use this fact below so no need to call `get_element_type()`

Reply:
Done.